agent/lib_aws_sdk_php.h

```diff
@@ -20,4 +30,71 @@ extern void nr_lib_aws_sdk_php_handle_version();
 extern void nr_lib_aws_sdk_php_add_supportability_service_metric(
     const char* service_name);
 
+#if ZEND_MODULE_API_NO >= ZEND_8_1_X_API_NO /* PHP8.1+ */
+/* Aside from service class and version detection, instrumentation is only
+ * supported with PHP 8.1+ */
+
+/*
+ * Purpose : Parses the QueueUrl to extract cloud_region, cloud_account_id, and
+ * destination_name.  The extraction sets all or none since the values from the
+ * same string and if it is malformed, it cannot be used.
+ *
+ * Params  : 1. The QueueUrl
+ *           2. message_params to set message_params.destination_name
+ *           3. cloud_attrs to set message_params.cloud_region,
+ * message_params.cloud_account_id
+ *
+ * Returns :
+ *
+ * Note: caller is responsible for
+ * freeing cloud_attrs.cloud_region, cloud_attrs.cloud_account_id, and
+ * message_params.destination_name
+ */
+extern void nr_lib_aws_sdk_php_sqs_parse_queueurl(
+    const char* sqs_queueurl,
+    nr_segment_message_params_t* message_params,
+    nr_segment_cloud_attrs_t* cloud_attrs);
+
+/*
+ * Purpose : Handle when an SqsClient initiates a command
+ *
+ * Params  : 1. NR_EXECUTE_ORIG_ARGS (execute_data, func_return_value)
+ *           2. segment : if we instrument the commandName, we'll need to end
+ * the segment as a message segment Returns :
+ *
+ */
+extern void nr_lib_aws_sdk_php_sqs_handle(nr_segment_t* segment,
+                                          NR_EXECUTE_PROTO);
+
+/*
+ * Purpose : The second argument to the Aws/AwsClient::__call function should be
+ * an array containing an array of argument name:value pairs. Given an argument
+ * name, this will return the value of the argument.
+ *
+ * Params  : 1. NR_EXECUTE_PROTO (execute_data, func_return_value)
+ *           2. arg_name: name of argument to extract from command arg array
+ *
+ * Returns : the value of the arg_name; NULL if does not exist
+ *
+ * Note: The caller is responsible for freeing the returned string value
+ *
+ */
+extern char* nr_lib_aws_sdk_php_get_command_arg_value(char* command_arg_name,
+                                                      NR_EXECUTE_PROTO);
+
+/*
+ * Purpose : Return the first argument to the Aws/AwsClient::__call function
+ * which should be the name of the command.
+ *
+ * Params  : 1. NR_EXECUTE_PROTO (execute_data, func_return_value)
+ *           2. command_name: name of the command being called
```

Comment:
Function's declaration doesn't have a second argument:
```suggestion
```

Reply:
Thanks for catching, descriptions needed updating after a refactor. Updated here:
https://github.com/newrelic/newrelic-php-agent/pull/1003/commits/c1f163f7c43f1b731e1b94a515b0ec47cf4383f5